monitoring/prober/rid/v2/test_isa_expiry.py

```diff
@@ -14,7 +14,7 @@
 
 
 def test_ensure_clean_workspace_v2(ids, session_ridv2):
-  resp = session_ridv2.get('{}/{}'.format(ISA_PATH, ids(ISA_TYPE)), scope=SCOPE_SP)
+  resp = session_ridv2.get('{}/{}'.format(ISA_PATH, ids(ISA_TYPE)), scope=SCOPE_DP)
```

Comment:
This is a bug in the OpenAPI -- Service Providers should be able to retrieve ISA information.  That change is now in `updated.yaml`, so the uastech/standards subrepo can be updated to [here](https://github.com/uastech/standards/commit/dd4016b09fc8cb98f30c2a17b5a088fb2995ab54) (and this should stay as `SCOPE_SP`)

Reply:
Now fixed